opentok/opentok-meet
# Add session options object to OTSession.init

#### What is this PR doing?
- Fix usage of `OTSession.init` to include session options. This was updated here https://github.com/maikthomas/OpenTok-Angular/blob/add-init-session-options/opentok-angular.js#L37
☝️ this is our current dependency: https://github.com/opentok/opentok-meet/blob/master/package.json#L27

#### How should this be manually tested?
- run locally
- open a room e.g. http://localhost:9000/testroom
- once connected open the webview composer app for that room: http://localhost:9000/testroom/webview-composer-app
- Check it works (you can try these steps with meet.tokbox.com and see that the page breaks with an error in the console.log saying.


#### What are the relevant tickets?

OPENTOK-48287
